lib/boxen/preflight/creds.rb

```diff
@@ -105,4 +122,28 @@ def fetch_from_env(thing)
     warn "Oh, looks like you've provided your #{thing} as environmental variable..."
     found
   end
+
+  def fingerprint
+    @fingerprint ||= begin
+      # See Apple technical note TN1103, "Uniquely Identifying a Macintosh
+      # Computer."
+      serial_number_match_data = IO.popen(
+        ["ioreg", "-c", "IOPlatformExpertDevice", "-d", "2"]
```

Comment:
@mikemcquaid - I saw your comment about supporting OS X 10.8. I have no clue whether this command is supported in 10.8, but I believe ioreg itself has been in OS X for quite some time.